engine/src/main/java/org/terasology/engine/TerasologyConstants.java

```diff
@@ -24,18 +24,52 @@
 import java.nio.file.Paths;
 
 /**
+ * This class contains various important file-related constants used by the Terasology engine.
  */
 public final class TerasologyConstants {
+    /**
+     * Name and extension of an entities data file.
```

Comment:
It's never used in the project :) We may never know! A lot of classes seem to redefine these constants because they didn't know this class existed. For example StoragePathProvider defines GLOBAL_ENTITY_STORE as "global.dat" as well as new world directory names and player data file names.